### PR TITLE
refresh

### LINKED ISSUE #4608: Page does not refresh went deleting a catalog entry
When you delete a catalog entry and click yes in the modal, nothing happens until the next refresh.

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

* Short description here *

closes #4608 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: changelogs/unreleased/4613-dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+change-type: patch
```
Comment (LauraStordeur) [inline changelogs/unreleased/4613-dependabot.yml:1]: why is this one in there?

Comment (LauraStordeur) [reply changelogs/unreleased/4613-dependabot.yml:1]: the merge tool will block the merge if there are multiple yml files :)

Comment (matborowczyk) [reply changelogs/unreleased/4613-dependabot.yml:1]: I made pull request from master, and something had to happen in between, as it's treated as something new, I made pull on master and there is that file on master

Comment (LauraStordeur) [reply changelogs/unreleased/4613-dependabot.yml:1]: yes, but if you pulled in master, this file should already be present and thus, not be in the diff 🤔

Comment (matborowczyk) [reply changelogs/unreleased/4613-dependabot.yml:1]: I am aware, but this file exist on master already https://github.com/inmanta/web-console/blob/master/changelogs/unreleased/4613-dependabot.yml

**Review by LauraStordeur — APPROVED** [review]

**Pull request closed without merging**